src/main/java/io/jenkins/plugins/step/WithGCPStep.java

```diff
@@ -103,10 +103,10 @@ public boolean start() throws Exception {
             }
 
             final var envVars = context.get(EnvVars.class);
-            envVars.put("GOOGLE_APPLICATION_CREDENTIALS", credentialsId);
-            final var projectId = extractProjectId(tempFile);
-            if (projectId != null) {
-                envVars.put("CLOUDSDK_CORE_PROJECT", projectId);
+            final var publicCredentialValues = extractPublicCredentialValues(tempFile);
+            if (publicCredentialValues != null) {
+                envVars.put("CLOUDSDK_CORE_ACCOUNT", publicCredentialValues.email);
```

Comment:
@el-chuck 
can you also update the [README](https://github.com/jenkinsci/pipeline-gcp-plugin/blob/master/README.md?plain=1#L25) file ?